### PR TITLE
Model Disutility of Care

### PR DESCRIPTION
**Pull request by vadi2**

### CODE REVIEW

File: input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Description: "A reoperation due to involved margins after primary surgery"
+* insert PublicationProfileRuleset
+* code from InvolvedMarginsReoperationType (required)
+* subject only Reference(BreastCancerPatient)
+* partOf MS
+  * ^short = "Original procedure that prompted the reoperation"
+  * ^definition = "A larger event of which this particular procedure is a component or step. In this case, the original procedure that prompted the reoperation."
+* performed[x] only dateTime 
+* performedDateTime and code and subject MS
+// TODO include a refence to the original procedure once treatment variables are modelled
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh:14]: also link to reasoncode and partof (CIC call feedback)

**Comment by MariekeSpanFurore** [comment]
I changed the complication to a condition in which both the type and impact of a complication fit fairly good. Furthermore I added the dueTo extention, which I can use once the treatment variables are finished. This is the same for the reoperation profile.

File: input/fsh/Terminology.fsh [hunk 2]
```diff
@@ -483,3 +483,68 @@ Description: "Valueset of the methods used to confirm recurrence of breast cance
...
+Description: "ValueSet of the impact of a complication experienced by the breast cancer patient"
+* SCT#240917005 "Interventional radiology"  
+* SCT#303577009 "Interventional debulking surgery" 
+* SCT#417005 "Hospital re-admission"
+* SCT#397945004 "Unexpected admission to intensive care unit"
+* SCT#274512008 "Drug therapy discontinued"
+* SCT#182879007 "Medication decreased"
+* SCT#419620001 "Death"
+* NullFlavor#UNK "unknown"
+// still need (single) codes for complication leading to prolonged hospitalization, discontinuing of treatment and none of the above.
```
Comment (vadi2) [inline input/fsh/Terminology.fsh:526]: Any reason not to use an ICHOM-specific codesystem here?

File: input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh [hunk 3]
```diff
@@ -0,0 +1,64 @@
+Profile: InvolvedMarginsReoperation
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/DisutilityOfCare.fsh:1]: How about https://www.hl7.org/fhir/extension-procedure-method.html for Reconstruction reoperation?

**Pull request merged**